acceptance/api/v1/applications_test.go

```diff
@@ -131,16 +135,58 @@ var _ = Describe("Apps API Application Endpoints", func() {
 		}, "5m").Should(Equal("True"))
 	}
 
+	uploadApplication := func(appName string) *models.UploadResponse {
+		uploadURL := serverURL + "/" + v1.Routes.Path("AppUpload", org, appName)
+		uploadPath := testenv.TestAssetPath("sample-app.tar")
+		uploadRequest, err := uploadRequest(uploadURL, uploadPath)
+		Expect(err).ToNot(HaveOccurred())
+		resp, err := env.Client().Do(uploadRequest)
+		Expect(err).ToNot(HaveOccurred())
+		bodyBytes, err := ioutil.ReadAll(resp.Body)
+		Expect(err).ToNot(HaveOccurred())
+
+		respObj := &models.UploadResponse{}
+		err = json.Unmarshal(bodyBytes, &respObj)
+		Expect(err).ToNot(HaveOccurred())
+
+		return respObj
+	}
+
+	// returns all the objects currently stored on the S3 storage
+	listS3Blobs := func() []string {
+		out, err := helpers.Kubectl("get", "secret",
+			"-n", "minio-epinio",
+			"tenant-creds", "-o", "jsonpath={.data.accesskey}")
+		Expect(err).ToNot(HaveOccurred(), out)
+		accessKey, err := base64.StdEncoding.DecodeString(string(out))
+		Expect(err).ToNot(HaveOccurred(), string(out))
+
+		out, err = helpers.Kubectl("get", "secret",
+			"-n", "minio-epinio",
+			"tenant-creds", "-o", "jsonpath={.data.secretkey}")
+		Expect(err).ToNot(HaveOccurred(), out)
+		secretKey, err := base64.StdEncoding.DecodeString(string(out))
+		Expect(err).ToNot(HaveOccurred(), string(out))
+
+		rand, err := randstr.Hex16()
+		Expect(err).ToNot(HaveOccurred(), out)
+		out, err = helpers.Kubectl("run", "-it",
+			"--restart=Never", "miniocli"+rand, "--rm",
+			"--image=minio/mc", "--command", "--",
+			"/bin/bash", "-c",
+			fmt.Sprintf("mc alias set minio http://minio.minio-epinio.svc.cluster.local %s %s 2>&1 > /dev/null && mc --quiet ls minio/epinio", string(accessKey), string(secretKey)))
```

Comment:
This might need some in-code docs. It took a while to understand that the alias command creates a shorthand name for the server which is subsequently used by the following `ls`. It did not help that the `ls` was nearly out of sight at the right border of the display.

Reply:
Done with this: https://github.com/epinio/epinio/pull/788/commits/110337c636e9b56d912f787ad46244a773c73a85 ?